modules/extensions-support/src/test/java/org/mule/module/extension/internal/runtime/connector/basic/BasicOperations.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.module.extension.internal.runtime.connector.basic;
+
+import org.mule.extension.annotation.api.Operation;
+
+public class BasicOperations
+{
+
+    @Operation
+    public void createAccount(Account account)
```

Comment:
actually do something with the account, and if possible, make this a non void operation. Maybe get it getOwner() and return the account's owner